robotd/devices.py

```diff
@@ -1,18 +1,18 @@
 """Actual device classes."""
 
 import os
-from pathlib import Path
 import random
 import struct
 import subprocess
 import time
+from pathlib import Path
 
-from sb_vision import Camera as VisionCamera, Vision, Token
 import serial
+from sb_vision import Camera as VisionCamera
+from sb_vision import Token, Vision
```

Comment:
Should these lines be merged?

`from sb_vision import Camera as VisionCamera, Token, Vision`

Reply:
Maybe, though this is what `isort` generated.

Reply:
hmm, looks like it's doing this split all over the place. Assuming it's standardised, i'm fine with it

Reply:
Yeah, I'm not sure I like it either. As noted in the PR I'd like to merge this soon and then sort out our preferences later. Especially in this case changing later is easy as you can just run `isort` on all the files and it'll convert over.